src/store/state.ts

```diff
@@ -124,15 +120,27 @@ export const emptyState: VQBState = {
   interpolateFunc: (x: string, _context: ScopeContext) => x,
 };
 
+/**
+ * @param state current application state
+ * @return the pipeline currently edited
+ */
+export function currentPipeline(state: VQBState) {
```

Comment:
Genuine question: any reason to have it as a function rather than directly being a getter on the store?

Reply:
I followed the model of the other functions defined in this file, below.
This function is imported into the getters file and used as a getter there.

We could also import the getter here and use it, it seems equivalent to me. The important thing is that we have the "logic" of getter a pipeline by its name only in one plcae?